src/QuickMarcEditor/QuickMarcEditWrapper.js

```diff
@@ -132,7 +153,41 @@ const QuickMarcEditWrapper = ({
 
         setHttpError(parsedError);
       });
-  }, [showCallout, refreshPageData, location, initialValues, instance, locations, marcType, mutator]);
+  }, [showCallout, refreshPageData, location, initialValues, instance, locations, marcType, mutator, numOfLinks]);
+
+  useEffect(() => {
```

Comment:
Hey Priyanka, does the snippet from this comment have any issues?
https://github.com/folio-org/ui-quick-marc/pull/417#discussion_r1044385960

Reply:
A callout was coming up on page load. Not sure of the reason.